src/main/java/com/artipie/rpm/Rpm.java

```diff
@@ -89,24 +86,34 @@
      */
     private final Digest digest;
 
+    /**
+     * Include filelists?
+     */
+    private final boolean filelists;
+
     /**
      * New Rpm for repository in storage.
      * @param stg The storage which contains repository
+     * @param filelists Include file lists in update
      */
-    public Rpm(final Storage stg) {
-        this(stg, StandardNamingPolicy.PLAIN, Digest.SHA256);
+    public Rpm(final Storage stg, final boolean filelists) {
```

Comment:
@carlosmiranda I believe it would be good to leave the original ctor that takes only `Storage` argument for backward compatibility. It would provide `false` value for `filelists`